### PR TITLE
OPR-10 OPR-11 OPR-12 OPR-13 OPR-14 OPR-15 sorting, more data, refacto…

### PR DESCRIPTION
**Pull request by m-pilarczyk**
…ring

### CODE REVIEW

File: app/components/atoms/ScrollableWrapper.js [hunk 1]
```diff
@@ -1,19 +1,10 @@
...
-
-  @media (max-width: ${breakpoints.tabletMd}px) {
-    overflow: hidden;
-    overflow-x: scroll;
-    width: 100%;
-  }
+  overflow: hidden;
+  overflow-x: auto;
+  width: 100%;
+  box-shadow: var(--box-shadow);
```
Comment (Meskat) [inline app/components/atoms/ScrollableWrapper.js]: It seems to be no longer scrollable...

File: app/components/organisms/ListView/styled.js [hunk 2]
```diff
@@ -1,7 +1,5 @@
 import styled from 'styled-components';
 
-export const ListViewWrapper = styled.section`
-  max-width: 90vw;
-`;
+export const ListViewWrapper = styled.section``;
```
Comment (Meskat) [inline app/components/organisms/ListView/styled.js]: The whole file can be deleted and swapped with section tag inside components

File: app/containers/App/styled.js [hunk 3]
```diff
@@ -7,13 +7,17 @@ export const AppContainer = styled.div`
...
   grid-gap: 16px;
-  grid-template-columns: 0.7fr 5fr 0.7fr;
+  grid-template-columns: 0.1fr 5fr 0.1fr;
   grid-template-rows: 70px 40px 1fr;
   grid-template-areas:
     'header header header '
     '. breadcrumbs . '
     '. container . ';
 
+  @media (min-width: ${breakpoints.desktopLn}px) {
```
Comment (Meskat) [inline app/containers/App/styled.js]: typo (desktopLn)

File: app/containers/BlockPage/selectors.js [hunk 4]
```diff
@@ -7,15 +6,13 @@ const selectBlockPageDomain = state => state.get('blockPage', initialState);
...
-
-    if (block.data.time) {
-      block.data.time = formatDate(block.data.time);
-    }
-
-    if (block.data.dividend_pay) {
-      block.data.dividend_pay =
-        block.data.dividend_pay === true ? 'true' : 'false';
-    }
+    block.prettyData = Object.assign(
```
Comment (Meskat) [inline app/containers/BlockPage/selectors.js]: why you are not using object destructuring?

Comment (m-pilarczyk) [reply app/containers/BlockPage/selectors.js]: changed

File: app/components/organisms/ListView/styled.js (hunk 2 above)
Comment (m-pilarczyk) [reply app/components/organisms/ListView/styled.js]: Removed

**Pull request merged**